assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,40 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {return};
+
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = param => param
+
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
-// add(1,2);
+
+let add = (param1, param2) => {
```

Comment:
```suggestion
const add = (param1, param2) => param1 + param2
```

I'm not sure if you had to break these into two lines and use a explicit return, but usually if you can get away with a one-liner and implicit return you tend to go with that, unless there were stipulations or you were told by the boss not too....

Reply:
Also, just as a general rule as JS emulates functional programming philosophies more and more the general rule is to stick to `const` and if you are going to def not keep it constant use `let` or as mostly in my case I just switch it to let if I end up running a READONLY error because I try to reassign...  , and for functions I can't think of a use case where it would be desirable to use `let` as you probably do not want it to reassign, not that its wrong, just you will tend to see `const` where variables wont change and the idea is that you eliminate a huge issue that was seen in JS when var was the only declaration type ... and that would be hard to deduce and figure out bugs do to the almost radical behavior of var and how it could be reassigned, hoisted, etc.

Mostly the few times you see let are for counter style initialization variables and some other points were reassigning can't be avoided.. Also, you can use const for objects and arrays and the items in arrays, or the key value pairs of objects can be mutated, you just can reassign the entire array to another one... i.e

`const arr = [1,2,4]`  you can do .push(), or arr[1] = 0 and change the items but you can't do something like this `arr = [3,4,55,2]` this will throw a read only // cannot reassign error.